.eslintrc.yml

```diff
@@ -6,7 +6,6 @@ env:
   amd: true
 extends:
   - 'eslint:recommended'
-  - 'plugin:react/recommended'
```

Comment:
Does this let us remove
https://github.com/interledger/rafiki/blob/9b30489300c90f456ce4ee37d28e39e31b9b1282/package.json#L37

Reply:
Probably...